controllers/external-accounts.js

```diff
@@ -87,10 +88,19 @@ const syncExternalAccountData = async (req, res) => {
           },
         };
       }
+
+      const unverifiedRoleId = config.get("discordUnverifiedRoleId");
+      const isUserHasUnverifiedRole = discordUser && discordUser.roles.includes(unverifiedRoleId);
+
+      if (isUserHasUnverifiedRole) {
+        markUsersVerifiedPromises.push(markUserVerified(discordUser.user.id));
+      }
+
```

Comment:
Can you please explain what the code does

Reply:
` const isUserHasUnverifiedRole = discordUser && discordUser.roles.includes(unverifiedRoleId);`

For the first condition, we check if a user is in Discord and has run the /verify command. For the second condition, we check if the user after running the /verify command has an unverified role.

Then, if we find such users we call a script to remove the unverified role by passing their discord id in the script.

Thanks